Source/Core/Common/Arm64Emitter.h

```diff
@@ -277,12 +277,12 @@ constexpr ARM64Reg EncodeRegToQuad(ARM64Reg reg)
   return static_cast<ARM64Reg>(reg | 0xC0);
 }
 
-enum ShiftType
+enum class ShiftType
 {
-  ST_LSL = 0,
-  ST_LSR = 1,
-  ST_ASR = 2,
-  ST_ROR = 3,
+  LogicalShiftLeft = 0,
```

Comment:
I think it'd still be informative as `LSL`, `LSR`, `ASR`, and `ROR` as the names for the shift types. These are what they're referred to as by ARM in official documentation, and the long-form makes code (imo) slightly more verbose to read and write.

Reply:
Given that you've made roughly half the commits touching Arm64Emitter in the last few years, I'm especially inclined to defer to your judgement. I've reverted the naming changes.

Said changes were inspired by my not knowing what the terms referred to until I did some googling. In order to help anyone else in a similar situation I've added some comments in the declaration with the full name, which at least in Visual Studio appear in the hover tooltip for the respective enum values.